### DISCUSSION TITLE
Inicialização de listas no método construtor

### DISCUSSION THREAD

**Discussion by luizgparaujo** [balta-io/forum#370] [Tira Dúvidas] [answered]
Estou realizando o curso de Fundamentos da Orientação a Objetos e durante a aula  Trabalhando com listas me surgiu uma dúvida:

No caso da lista **Modules**, a recomendação para inicialização da lista é sempre no **método construtor**. Isso é uma boa prática, apenas recomendação ou o jeito certo de se fazer? Pergunto isso pois no projeto em que trabalho, geralmente inicializamos a lista direto na propriedade.

![image](https://github.com/balta-io/forum/assets/52684053/15b4ca65-16f1-4f4e-8aa6-71ed8891444a)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @luizgparaujo, como vai?

A recomendação é sempre inicializar as listas, seja no construtor ou nas propriedades, não importa.

**Reply by luizgparaujo** [reply]
Show, obrigado @andrebaltieri !
